src/bun.js/bindings/bindings.cpp

```diff
@@ -2226,6 +2213,7 @@ JSC::EncodedJSValue SystemError__toErrorInstanceWithInfoObject(const SystemError
 
     JSC::JSValue options = JSC::jsUndefined();
     JSC::JSObject* result = JSC::ErrorInstance::create(globalObject, JSC::ErrorInstance::createStructure(vm, globalObject, globalObject->errorPrototype()), message, options);
+    RETURN_IF_EXCEPTION(scope, {});
```

Comment:
```suggestion
    WTF::String message = makeString("A system error occurred: "_s, syscallString, " returned "_s, codeString, " ("_s, messageString, ")"_s);
    JSC::JSObject* result = JSC::ErrorInstance::create(vm, JSC::ErrorInstance::createStructure(vm, globalObject, globalObject->errorPrototype()), message, {});
```

let's also use the one that doesn't throw here